src/integration-test/integration/named-groups/NamedGroupsClient.test.ts

```diff
@@ -0,0 +1,133 @@
+    for (const group of groups.groups) {
+      expect(["GroupOne 🚀", "GroupTwo 👻", "GroupThree"]).to.include(group.displayName);
+    }
+  });
+
+  it("NamedGroups - Get top minimal groups", async () => {
+    const topGroups = await namedGroupsClient.getNamedGroups(accessToken, iTwinId, PreferReturn.Minimal, 2);
+    expect(topGroups.groups.length).to.equal(2);
+    topGroups.groups.forEach((group) => {
+      expect("metadata" in group).to.be.false;
+    });
+  });
+
+  it("NamedGroups - Get top representation groups", async () => {
+    const topGroups = await namedGroupsClient.getNamedGroups(accessToken, iTwinId, PreferReturn.Representation, 2);
+    expect(topGroups.groups.length).to.equal(2);
+    topGroups.groups.forEach((group) => {
+      expect(group).to.have.property("metadata");
+    });
+  });
+
+  it("NamedGroups - Get pages of minimal groups", async () => {
+    const groupsIterator = namedGroupsClient.getNamedGroupsIterator(accessToken, iTwinId, PreferReturn.Minimal, 2);
+    let flag = false;
+    for await (const groupsPage of groupsIterator.byPage()) {
+      flag = true;
+      for (const group of groupsPage) {
+        expect(["GroupOne 🚀", "GroupTwo 👻", "GroupThree"]).to.include(group.displayName);
+        expect("metadata" in group).to.be.false;
+      }
+    }
+    expect(flag).to.be.true;
+  });
+
+  it("NamedGroups - Get pages of representation groups", async () => {
+    const groupsIterator = namedGroupsClient.getNamedGroupsIterator(accessToken, iTwinId, PreferReturn.Representation, 2);
+    let flag = false;
+    for await (const groupsPage of groupsIterator.byPage()) {
+      flag = true;
+      for (const group of groupsPage) {
+        expect(["GroupOne 🚀", "GroupTwo 👻", "GroupThree"]).to.include(group.displayName);
+        expect(group).to.have.property("metadata");
+      }
+    }
+    expect(flag).to.be.true;
+  });
+
+  it("NamedGroups - Create and Delete", async () => {
+    const group = await namedGroupsClient.createNamedGroup(accessToken, {
+      iTwinId,
+      displayName: "GroupToDelete",
+      query: "select * from biscore.element limit 10",
+    });
+
+    expect(group).not.be.undefined;
+    expect(group.displayName).to.deep.equal("GroupToDelete");
+
+    const response = await namedGroupsClient.deleteNamedGroup(accessToken, group.id);
+    expect(response).not.be.undefined;
+    expect(response.status).to.be.equal(204);
+  });
+
+  it("NamedGroups - Update", async () => {
+    const updatedGroupOne: NamedGroupUpdate = {
+      displayName: "UpdatedGroupOne",
+      description: "Updated description for group one",
+    };
+
+    const updatedGroup = await namedGroupsClient.updateNamedGroup(accessToken, groupOne.id, updatedGroupOne);
+
+    expect(updatedGroup.displayName).not.be.undefined;
+    expect(updatedGroup.displayName).to.deep.equal(updatedGroupOne.displayName);
+    expect(updatedGroup.description).to.deep.equal(updatedGroupOne.description);
+  });
+
```

Comment:
Remove trailing empty lines.

Reply:
`ESLint` `eol-last` rule.

Reply:
Oh my bad. I thought this was at the end of the file.

Reply:
Fixed